### ISSUE TITLE
UI: Wallet history is displaying only the last 100 transactions

### ISSUE THREAD

**Issue by turcol** [block-core/blockcore#164]
The history is only displaying the last 100 transactions, and there is no way to navigate to the oldest transactions

**Comment by dangershony** [comment]
yes good point we need a button at the bottom to load more perhaps?

**Comment by thecrypt0hunter** [comment]
Ideally we add pagination so we can navigate through the tx history
